Specs/createScene.js

```diff
@@ -16,9 +16,14 @@ define([
     "use strict";
 
     function createScene(options) {
+        options = defaultValue(options, {});
+
+        var canvas = defaultValue(options.canvas, createCanvas());
```

Comment:
This should be `defined(options.canvas) ? options.canvas : createCanvas()`. The way it's currently written, if `options.canvas` is specified, we add an extra canvas to the document and it never gets removed (which could technically interfere with future specs).